configs/atlas-xcache/config.d/45-atlas-xcache-plugin.cfg

```diff
@@ -0,0 +1,13 @@
+# overwriting defaults to suits ATLAS needs
+
+if defined ?~XC_BLOCKSIZE
+   set eBLOCKSIZE=$XC_BLOCKSIZE
+else
+   set eBLOCKSIZE=1M
+fi
+
+if defined ?~XC_PREFETCH
+   set ePREFETCH=$XC_PREFETCH
+else
+   set ePREFETCH=0
+fi
```

Comment:
```suggestion
fi

# The size of the blocks in cache; all reads to the data federation
# will be rounded along these boundaries.
pfc.blocksize $eBLOCKSIZE
# Don't prefetch anything
pfc.prefetch $ePREFETCH
```

I don't think this will actually work since the variables get used in an earlier `40-*cfg` in `pfc.blocksize` and `pfc.prefetch`. You may be able to just add the above but I never know with XRootD configuration